Sources/XMTPiOS/Frames/FramesClient.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alex Risch on 3/28/24.
+//
+
+import Foundation
+import LibXMTP
+
+public typealias FrameActionBody = Xmtp_MessageContents_FrameActionBody
+public typealias FrameAction = Xmtp_MessageContents_FrameAction
+
+class FramesClient {
```

Comment:
Do you plan on this being extended? If not probably safe to make the client final.

```suggestion
public final class FramesClient {
```

Reply:
I imagine someone _could_ want to extend it, not sure of the why though